lips/lip-10.md

```diff
@@ -0,0 +1,221 @@
+  * Each VASP can create the user-id however they see fit - user selected, auto-generated, etc. and these are not required to be unique across different VASPs because the domain provides disambiguation across different VASPs. This user-id is a parallel to the user ID in an email address
+* **libra-id-domain** identifies the VASP:
+  * Case insensitive
+  * Regex: `^[a-zA-Z0-9][a-zA-Z0-9-]+$`
+  * Maximum length: 63 characters
+  * libra-id-domains are published on the chain
+  * Each libra-id-domain is uniquely associated with single VASP
+
+## Libra ID domains
+Each Libra ID domain is associated with a single VASP. The source of truth for this association is the blockchain - a special LibraIdDomains resource must be published into VASP for it to expose its LibraID domain.
+
+```
+resource struct LibraIdDomains {
+    domains: vector<LibraIdDomain>,
+}
+
+struct LibraIdDomain {
+    domain: vector<u8>,     // Utf-8 encoding
+    wallet_uri: vector<u8>, // ASCII encoding (use url-escape for non-ASCII characters)
+}
+```
+
+* The `LibraIdDomains` resource can only be published into an account with `Role == PARENT_VASP_ROLE_ID`.
+* The `LibraIdDomains` contains a list of `LibraIdDomain` structs that can be associated with a VASP. As such, it is possible to register more than one Libra ID Domain for a given VASP. We are providing this opportunity to support possible situations when two VASPs merge into one, or when a company wants to provide multiple wallet apps, while having only a single VASP parent account on the chain.
+* Only special Treasury Compliance account (address `0xB1E55ED`) can manipulate LibraIdDomains resource:
+  * Only TC account can create and publish LibraIdDomains resource
+  * Only TC account can add, remove or update an LibraIdDomain within LibraIdDomains resource
+* In order to register a Libra Id domain, VASP needs to submit a request to Libra Association. LA will perform certain checks (out of scope of this document) before they will issue an on-chain transaction to register a Libra Id Domain. The checks that LA will perform will make sure that VASPs do not claim irrelevant Libra Id Domains. LA submission will also ensure that Libra Id Domains are unique.
+* It should also be possible (and probably going to be most common) for a new VASP to submit their desired Libra ID domain(s) along with the request to create a `ParentVASP` account.
+* An entity (potentially the Association) may at some point choose to expose an open source application(*Indexer*) that will allow indexing available Libra IDs and could provide a convenient REST API for applications to fetch information about the domains. The API of such an indexer is out of scope of this RFC.
+
+# Utilization of LibraID during payments
+One of the main use cases for Libra ID is transferring money from one user to another across different wallets.
+
+<img alt="Payment flow" src={useBaseUrl('img/libra-id-flow.png')} />
+
+## UI flow
```

Comment:
This is your story. The story lacks a lot of details that are specified below, such as the receiver side experience.